docs/config/aws_template.md

```diff
@@ -0,0 +1,440 @@
+
+=== "Deployment result"
+
+    Ingest and VAP pipelines can be set up to run on S3 event triggers or on a cron schedule.
+    
+    ![User friendly overview](../figures/aws/user_friendly_overview.png)
+
+=== "Stack resources"
+
+    The following resources will be used or setup during the AWS deployment. 
+    
+    ![AWS Template](../figures/aws/stack_resources.png)
+
+=== "CodeBuild steps"
+
+    The following image shows how code changes are deployed to AWS and the steps that make up the AWS CodeBuild process
+    to update or create lambda functions and containers for each pipeline.
+
+    ![Image](../figures/aws/code_build.png)
+
+## Prerequisites
+
+### Create code repos
+
+Make sure that you have two repositories in your GitHub organization/account created from the following templates:
+
+* **<https://github.com/tsdat/pipeline-template>**
+* **<https://github.com/tsdat/aws-template>**
+
+If you are using an existing `pipeline-template` repository, make sure that the `requirements.txt` file specifies a
+`tsdat` version of at least `tsdat==0.7.1`. The AWS build will not work with earlier versions of `tsdat`.
+
+Clone these repos to the same parent folder on your computer.
+
+### Install Docker
+
+We use a Docker container with VSCode to make setting up your development environment a snap.  We assume users have a
+basic familiarity with Docker containers. If you are new to Docker, there are many free online tutorials to get you
+started.
+
+!!! note
+
+    Docker Desktop can be flaky, especially on Windows, and it requires a license so we recommend not using it. Instead,
+    we are providing alternative, non-Docker Desktop installation instructions for each platform. The Docker Desktop
+    install is easier and requires fewer steps, so it may be fine for your needs, but keep in mind it may crash if you
+    update it (requiring a full uninstall/reinstall, and then you lose all your container environments).
+
+    === "Windows Users"
+
+        * [Install Docker on wsl2](https://github.com/clansing/docs/blob/main/windows-docker-wsl2.md)
+        * [Install Docker Desktop](https://docs.docker.com/desktop/install/windows-install/)
+
+    === "Mac Users"
+
+        * [Use Docker/Colima](https://dev.to/elliotalexander/how-to-use-docker-without-docker-desktop-on-macos-217m)
+        * [Install Docker Desktop](https://docs.docker.com/desktop/install/mac-install/)
+
+    === "Linux Users"
+
+        * [Install Docker](https://docs.docker.com/engine/install/ubuntu/)
+        * [Install Docker Desktop](https://docs.docker.com/desktop/install/linux-install/)
+
+We also recommend [installing VSCode](https://code.visualstudio.com/download) and using the
+[ms-vscode-remote.vscode-remote-extensionpack](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.vscode-remote-extensionpack)
+extension, which includes support for editing code in Docker Containers.
+
+## Development Environment
+
+### Open your `aws-template` repo in VSCode
```

Comment:
This level-3 section header is not rendering very differently from the normal body text and the level-2 header.  This makes it hard for me to find the section I'm looking for when I'm scanning the page.  Is there a way to make the h3 and h3 styles bolder and make the h2 header a different color?  Something like this:

![image](https://github.com/tsdat/tsdat/assets/1616956/7f5ec996-5941-46a1-96c1-fef50fd5e42c)

Reply:
Yes, looks like this is possible. I made it look like this:

<img width="1367" alt="image" src="https://github.com/tsdat/tsdat/assets/24307537/174c9609-b1d3-4cc0-a3d1-e64a3ebf1a7a">